sycl/doc/extensions/SRGB/srgb_support.asciidoc

```diff
@@ -0,0 +1,163 @@
+= srgb_support
+
+// This section needs to be after the document title.
+:doctype: book
+:toc2:
+:toc: left
+:encoding: utf-8
+:lang: en
+
+:blank: pass:[ +]
+
+// Set the default source code type in this document to C,
+// for syntax highlighting purposes.
+:language: c
+
+// This is what is needed for C++, since docbook uses c++
+// and everything else uses cpp.  This doesn't work when
+// source blocks are in table cells, though, so don't use
+// C++ unless it is required.
+//:language: {basebackend@docbook:c++:cpp}
+
+
+
+== Name Strings
+
+`srgb_support`
+
+== Contact
+
+Chris Perkins, Intel (chris 'dot' perkins 'at' intel 'dot' com)
+
+== Contributors
+
+// spell-checker: disable
+Chris Perkins, Intel +
+// spell-checker: enable
+
+== Notice
+
+Copyright (c) 2021 Intel Corporation.  All rights reserved.
+
+== Status
+
+Working Draft
+
+This is a preview extension specification, intended to provide early access to a feature for review and community feedback.
+When the feature matures, this specification may be released as a formal extension.
+
+Because the interfaces defined by this specification are not final and are subject to change they are not intended to be used by shipping software products.
+If you are interested in using this feature in your software product, please let us know!
+
+== Version
+
+Built On: {docdate} +
+Revision: R
+
+== Dependencies
+
+This extension is written against the SYCL 1.2.1 specification, Revision v1.2.1-6.
+
+
+== Overview
+
+sRGB isn't an image data format, per se. It is a linearization of RGB color values that adjusts the color to be better matched with a particular hardware medium. A more detailed explanation may be found here: https://software.intel.com/content/www/us/en/develop/articles/using-opencl-20-srgb-image-format.html
+
+OpenCL provides automatic linearization of RGB values into sRGB color space. OneAPI now exposes that OpenCL support for read_only images. 
+
+== New ImageChannelOrder Enums
+
+sRGB values have been added to the `image_channel_order` enumeration. The `image_channel_order::srgba` value allows images to be read with sRGB linearization of the color values. This value can be passed to `image` constructors, with some limitations that are outlined below.
+
+[source]
+----
+image_channel_order::srgba
```

Comment:
It looks like the implementation actually adds three new enumerated values:

* srgb
* srgbx
* srgba

Should they all be listed here?  I presume there should be some description explaining what each one means?

Reply:
added the others to the docs.